$NetBSD: patch-ae,v 1.3 2013/08/22 11:09:46 christos Exp $

* check sizeof time_t.
* check existence of sigsend().
* check gethosybyname_r declaration

--- configure.in.orig	2013-07-11 16:39:36.000000000 +0300
+++ configure.in	2013-08-22 11:15:43.000000000 +0300
@@ -861,6 +861,7 @@
 AC_CHECK_LIB(nsl_r, gethostbyname_r, [],
   AC_CHECK_LIB(nsl, gethostbyname_r))
 AC_CHECK_FUNC(gethostbyname_r)
+AC_CHECK_DECLS([gethostbyname_r],[],[],[#include <netdb.h>])
 
 AC_SEARCH_LIBS(setsockopt, socket)
 # This may get things to compile even if bind-8 is installed
@@ -1902,6 +1903,7 @@
 then
   AC_MSG_ERROR("MySQL needs a off_t type.")
 fi
+AC_CHECK_SIZEOF(time_t, 4)
 
 dnl
 dnl check if time_t is unsigned
@@ -2004,7 +2006,7 @@
 paths.h semaphore.h)
 
 # Already-done: strcasecmp
-AC_CHECK_FUNCS(lstat putenv select setenv setlocale strcoll tcgetattr)
+AC_CHECK_FUNCS(lstat putenv select setenv setlocale sigsend strcoll tcgetattr)
 
 AC_STAT_MACROS_BROKEN
 MYSQL_SIGNAL_CHECK
